### DISCUSSION TITLE
Tags for classical music

### DISCUSSION THREAD

**Discussion by MRodneyMcKay** [navidrome/navidrome#2953] [Ideas]
It would be nice if we could get a working system for works, movements and movement names just like os qobuz's webplayer.
Is there any current effort on this topic?

**Comment by sramocki** [comment]
There's an open pull request [here.](https://github.com/navidrome/navidrome/pull/2601)

**Reply by MRodneyMcKay** [reply]
Yes, and the last commit was on 2023 november, it seems abbandoned.
I do not know why. If it was because of the lack of interest, my post proves that there is some :D

**Reply by sramocki** [reply]
I'm definitely extremely interested as well. I believe the PR is being held up by changes to the scanner

https://github.com/navidrome/navidrome/pull/2709

**Reply by certuna** [reply]
> Yes, and the last commit was on 2023 november, it seems abbandoned.
> 
> I do not know why. If it was because of the lack of interest, my post proves that there is some :D

I wrote this, indeed we need to wait for a) the new scanner and ideally also b) formal support for these fields in the OpenSubsonic API

**Reply by amorphobia** [reply]
I wander how it going in the OpenSubsonic API? Any issues to track it?
